Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.7-slim as base
```

Comment:
Can we rename this to minioloader.Dockerfile? Since we also have `ybloader.Dockerfile`?